src/axom/spin/tests/spin_bvh.cpp

```diff
@@ -1372,11 +1373,11 @@ void bvh_one_bbox_2d(BVHType& bvh,
 
   // Initialize the BVH with the points.
   axom::IndexType npts = none ? 0 : points.size();
-  //axom::Array<BoxType> bboxes;
-  axom::Array<BoxType> bboxes;
+  BoxType* bboxes = axom::allocate<BoxType>(npts + 1,
+                                            axom::execution_space<ExecSpace>::allocatorID());
```

Comment:
The following should also work:
```cpp
auto allocID = axom::execution_space<ExecSpace>::allocatorID();
axom::Array<BoxType> bboxes(npts, npts, allocID);

for(axom::IndexType i = 0; i < npts; i++)
{
    bboxes[i] = BoxType(points[i]);
}
```

Reply:
i.e. you can construct and allocate an array of the desired size using a desired allocator.